src/libraries/Microsoft.Extensions.Options.DataAnnotations/src/Microsoft.Extensions.Options.DataAnnotations.csproj

```diff
@@ -1,13 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Description>Provides additional DataAnnotations specific functionality related to Options.</Description>
     <TargetFrameworks>netstandard2.0;$(DefaultNetCoreTargetFramework)</TargetFrameworks>
     <TargetFrameworks Condition="'$(DotNetBuildFromSource)' == 'true'">$(DefaultNetCoreTargetFramework)</TargetFrameworks>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <PackageTags>aspnetcore;validation;options</PackageTags>
```

Comment:
Where do these "PackageTags" go now? If we are removing them here, where should it go?

Reply:
If we wanted to maintain them they'd go in the pkgproj, but we'd need to fix this: https://github.com/dotnet/arcade/blob/ed8c23ff7c4f4988a754ff9afc626571f16e3382/src/Microsoft.DotNet.Build.Tasks.Packaging/src/build/Packaging.targets#L52

Reply:
Honestly I don't think they are worth preserving, but I'm happy to hear otherwise if folks feel strongly.

Reply:
@anurse @bradygaster - how much effort/thought went into the existing tags?

I agree we can easily dropp `aspnetcore` from these. But I'm not so sure about the other tags.

Reply:
Thought? Almost none ;). Feel free to change them at-will.